crypto/txscript/src/data_stack.rs

```diff
@@ -191,7 +191,7 @@ impl OpcodeData<i32> for Vec<u8> {
     #[inline]
     fn deserialize(&self) -> Result<i32, TxScriptError> {
         let res = OpcodeData::<i64>::deserialize(self)?;
-        Ok(res.clamp(i32::MIN as i64, i32::MAX as i64) as i32)
+        Ok(res.clamp(i32::MIN as i64, i32::MAX as i64) as i32) // todo rid of clamp?
```

Comment:
Can you explain shortly the reasoning?
Maybe something like "TODO: Consider getting rid of clamp, since the call to `deserialize` should return an error if the number is not in the i32 range (this should be done with proper testing)"